exporters/otlp/include/opentelemetry/exporters/otlp/otlp_exporter.h

```diff
@@ -15,6 +15,11 @@ struct OtlpExporterOptions
 {
   // The endpoint to export to. By default the OpenTelemetry Collector's default endpoint.
   std::string endpoint = "localhost:4317";
+  // By default when false, uses grpc::InsecureChannelCredentials(); If true,
+  // uses ssl_credentials_cacert_path
+  bool use_ssl_credentials = false;
+  // ssl_credentials_cacert_path specifies path to .pem file to be used for SSL encryption.
+  std::string ssl_credentials_cacert_path = "";
```

Comment:
Could we add an option to add the certificates as strings, and not as a path to a file containing the certificates?

That would make us more flexible, as there are many instances where certificates aren't available on the file system but only in memory.

Reply:
Would you like it as additional option to file system path or you would like to get rid of file system path? If it is the latter, are all reviewers okay with the change?

Reply:
I think this is good idea to read cert from variable instead of file, that way exporter don't need to deal with file related error scenarios. Let application read the cert from file correctly, and inject it to exporter.

Reply:
In latest commit, added one more option for certificate as a string however, let me know if file option needs to be removed - for our current application needs, we were using file system paths and it is working correctly.